roles/pas/defaults/main.yml

```diff
@@ -21,6 +21,7 @@ pas_db_schema: 'public'
 pas_db_table_prefix: 'craft_'
 pas_security_key: 'abc123'
 pas_default_site_url: 'https://localhost:8484'
+craft_version: 3
```

Comment:
The new version we are trying is Craft 4.

Reply:
Would you like the default to be Craft 4, since that's what we're moving towards? We wanted it to be backwards compatible for production and staging, since we don't have Craft 4 working yet.